CoreErrorReportView/release/modules/ext/docs/CoreErrorReportView/src/au/gov/asd/tac/constellation/views/errorreport/error-report.md

```diff
@@ -0,0 +1,56 @@
+# Error Report
```

Comment:
It would be good to add the table at the top of the page with shortcuts/how to navigate to the view. It would also be useful to add a help button to the actual view as well.